keyvault/src/main/java/com/microsoft/identity/internal/test/keyvault/api/DeletedCertificatesApi.java

```diff
@@ -150,7 +150,7 @@ public DeletedCertificateBundle getDeletedCertificate(String certificateName, St
      */
     public ApiResponse<DeletedCertificateBundle> getDeletedCertificateWithHttpInfo(String certificateName, String apiVersion) throws ApiException {
         com.squareup.okhttp.Call call = getDeletedCertificateValidateBeforeCall(certificateName, apiVersion, null, null);
-        Type localVarReturnType = new TypeToken<DeletedCertificateBundle>(){}.getType();
```

Comment:
This and similar changes seem not related to TypeToken.getParameterized and instead call `[TypeToken.get](https://github.com/google/gson/blob/9414b9b3b61d59474a274aab21193391e5b97e52/gson/src/main/java/com/google/gson/reflect/TypeToken.java#L302)`

Reply:
Yes, I have updated the PR title. The idea is to remove the use of `new TypeToken`